lib/charms/opensearch/v0/opensearch_base_charm.py

```diff
@@ -1030,17 +1030,6 @@ def _stop_opensearch(self, *, restart=False) -> None:
         self.peers_data.delete(Scope.UNIT, "started")
         self.status.set(WaitingStatus(ServiceStopped))
 
-        # 3. Remove the exclusions
-        if not restart:
-            try:
-                self.opensearch_exclusions.delete_current()
```

Comment:
We do not remove it here, but rather at "start_openseach" method now

Reply:
FYI, the `delete_current()` is already called on `_post_start_init`. No changes needed there then